src/exporters/EmbraceLogExporter/OTLPFetchLogExporter.ts

```diff
@@ -4,8 +4,8 @@ import type {
   ReadableLogRecord,
 } from '@opentelemetry/sdk-logs';
 import { BaseFetchExporter } from '../BaseFetchExporter/index.js';
-import { createOtlpBrowserFetchExportDelegate } from '../otlpBrowserFetchExportDelegate.js';
-import type { OtlpFetchExporterConfig } from '../types.js';
+import type { OtlpFetchExporterConfig } from '../index.js';
+import { createOtlpBrowserFetchExportDelegate } from '../index.js';
```

Comment:
not sure if possible to enforce with the linter but for something like this would it be preferred to have it like this?

```suggestion
import { createOtlpBrowserFetchExportDelegate, type OtlpFetchExporterConfig } from '../index.js';
```

Reply:
I like having typescript-specific lines as isolated statements. It makes it a lot easier for bundlers, node, and other tools to just ignore it or tree shake it when not dealing with types. Also, now that we are no longer importing from a "types" file, it is clear in its intent if the whole line is prefixes with "type".  Does it makes sense? 

Changing it is just a linter rule, yes, but I made it this way because I liked it more 😅

Reply:
it is also less verbose when you have mora than one type imported, you don't need to repeat the `type`

Reply:
np, yep let's go with this way then